package.json

```diff
@@ -15,10 +15,12 @@
     "express-validator": "^5.3.1",
     "i": "^0.3.6",
     "mysql": "^2.16.0",
+    "nodemailer": "^6.0.0",
     "npm": "^6.4.1",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
-    "prop-types": "^15.6.2"
+    "prop-types": "^15.6.2",
```

Comment:
i dont think you added this but we dont need prop-types in the server package.json

Reply:
Also you should rerun npm install in the client to get the package-lock.json file back and commit it

Reply:
kk done